lib/auxiliary/id_gen.rb

```diff
@@ -16,9 +16,12 @@ def initialize(range, timeout)
 
     def load_data
       ids = Array.new(@range.last + 1 - @range.first, true)
-      @conn.execute 'CREATE TABLE IF NOT EXISTS ActiveIds('\
-                  'IdNumber INT PRIMARY KEY NOT NULL, PID INT NOT NULL,'\
-                  'Time TIMESTAMP NOT NULL)'
+      @conn.execute <<~SQL
+        CREATE TABLE IF NOT EXISTS ActiveIds(
+        IdNumber INT PRIMARY KEY NOT NULL, PID INT NOT NULL,
```

Comment:
I think you should add more indents in braces because you would do so in Ruby while this is in SQL.